partner_delivery_schedule/tests/test_partner_delivery_schedule.py

```diff
@@ -3,21 +3,14 @@
 from odoo.exceptions import ValidationError
 from odoo.tests import Form, TransactionCase
 
+from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
+
 
 class TestStockPikingReturnRefundOption(TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.env = cls.env(
-            context=dict(
-                cls.env.context,
-                mail_create_nolog=True,
-                mail_create_nosubscribe=True,
-                mail_notrack=True,
-                no_reset_password=True,
-                tracking_disable=True,
-            )
-        )
+        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
```

Comment:
@victoralmau FYI you can also inherits from odoo.addons.base.tests.common.BaseCommon to avoid this line at all